apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/dashboard/_components/Layout.tsx

```diff
@@ -1,100 +1,243 @@
 'use client'
 
+import { useState } from 'react'
+
 import { EvaluationDto } from '@latitude-data/core/browser'
 import {
   BlankSlateStep,
-  BlankSlateStepSkeleton,
   BlankSlateWithSteps,
   Button,
-  Icon,
-  TableBlankSlate,
+  cn,
+  FakeProgress,
+  Modal,
   Text,
+  Tooltip,
   useCurrentCommit,
-  useCurrentDocument,
   useCurrentProject,
+  useToast,
 } from '@latitude-data/web-ui'
-import { connectEvaluationsAction } from '$/actions/evaluations/connect'
+import { createEvaluationFromPromptAction } from '$/actions/evaluations/createFromPrompt'
+import { generateSuggestedEvaluationsAction } from '$/actions/evaluations/generateSuggestedEvaluations'
+import { useCurrentDocument } from '$/app/providers/DocumentProvider'
+import TypewriterText from '$/components/TypewriterText'
 import useLatitudeAction from '$/hooks/useLatitudeAction'
 import { useNavigate } from '$/hooks/useNavigate'
 import { ROUTES } from '$/services/routes'
 import useEvaluations from '$/stores/evaluations'
-import useSuggestedEvaluations, {
-  SuggestedEvaluation,
-} from '$/stores/suggestedEvaluations'
-import Link from 'next/link'
 
 import BatchEvaluationsTable from './BatchEvaluationsTable'
 
 function SuggestedEvaluations() {
   const document = useCurrentDocument()
   const { project } = useCurrentProject()
   const { commit } = useCurrentCommit()
-  const { data: suggestions, isLoading } = useSuggestedEvaluations(
-    document.content,
-  )
   const navigate = useNavigate()
   const { mutate } = useEvaluations()
-  const { execute, isPending } = useLatitudeAction(connectEvaluationsAction)
-  const onConnect = async (suggestion: SuggestedEvaluation) => {
-    const [data] = await execute({
+  const { execute: generateEvaluation, isPending: isGenerating } =
+    useLatitudeAction(generateSuggestedEvaluationsAction, {
+      onSuccess: () => {
+        // do nothing
```

Comment:
No need no?

Reply:
yeah it's needed to avoid the default toast which i don't want in this case